jdaviz/app.py

```diff
@@ -2021,8 +2021,7 @@ def _on_data_deleted(self, msg):
 
         self._clear_object_cache(msg.data.label)
 
-    @staticmethod
```

Comment:
I wonder why this was static...

Reply:
I suspect just because it never actually used `self` 🤷‍♂️